practices/optimizations.md

```diff
@@ -0,0 +1,55 @@
+---
+slug: optimizations
+stages:
+  - development
+short_description: Optimization is the process of modifying a software system to make some aspect of it work more efficiently or use fewer resources
+tags:
+  - optimizations
+  - code coverage
+keywords:
+  - code  - quality
+  - merge
+  - testing
+  - codebase
+---
+
+# Optimizations
+
+**TL;DR**
+
+Optimization is an act, process, or methodology of making something (such as a design, system, or decision) as fully perfect, functional, or effective as possible
+
+## What Is an Optimization
+The code optimization in the synthesis phase is a program transformation technique, which tries to improve the intermediate code by making it consume fewer resources (i.e. CPU, Memory) so that faster-running machine code will result.
```

Comment:
Please, rewrite this with your own words: _The code optimization in the synthesis phase is a program transformation technique, which tries to improve the intermediate code by making it consume fewer resources (i.e. CPU, Memory) so that faster-running machine code will result._ Its too similar to https://www.tececorp.com/code-optimization/

Reply:
done